pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2693,6 +2792,22 @@ int BPF_KPROBE(trace_do_exit)
     return events_perf_submit(&data, DO_EXIT, code);
 }
 
+SEC("kprobe/security_file_ioctl")
+int BPF_KPROBE(trace_security_file_ioctl)
```

Comment:
The name of the program is mutable, so I think you can call it `trace_tracee_trigger_event` or something similar.